Tests/src/artofillusion/SceneTest.java

```diff
@@ -74,16 +74,15 @@ public void testCreateScene()
    * Check that object created, added to scene and contains default tracks...
    */
   @Test
-    public void testAddObject1() {
+    public void testAddObjectAsNewObject3DAndCoordinates() {
 
       int sceneObjects = scene.getNumObjects();
 
       scene.addObject(new Cube(1d,1d,1d), new CoordinateSystem(), "Cube", (UndoRecord)null);
       ObjectInfo so = scene.getObject("Cube");
       
       assertEquals(++sceneObjects, scene.getNumObjects());
-      
-      assertNotNull(so);
+
       assertEquals("Cube", so.getName());
```

Comment:
Thanks for your update. I think you missed my point about redundant asserts, though. I'm specifically referring to lines like:

```
assertEquals("Cube", so.getName());
```

You just created this cube object, so there should be no need to check on its name.